### ISSUE TITLE
How to perform ensembling?

### ISSUE THREAD

**Issue by ChaoningZhang** [valeoai/ADVENT#20]
Very insightful paper. I have one question.
In the paper, the ensembling of EntMin and EntAdv achieves better performance.
How did you perform the ensembling? 
Did you calculate the average of the probability map (after the softmax) or logit map (before the softmax) of the two models?

Thanks for your reply in advance.
